### PR TITLE
Update foreman-remote-execution to 3.0.1

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: plugins/ruby-foreman-tasks/debian/control [hunk 1]
```diff
@@ -2,14 +2,14 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.17.0~rc1), foreman-sqlite3 (>= 1.17.0~rc1), foreman-assets
+Build-Depends: debhelper, foreman (>= 1.25.0~rc1), foreman-sqlite3 (>= 1.25.0~rc1), foreman-assets
```
Comment (mmoll) [inline plugins/ruby-foreman-tasks/debian/control]: Wrong file? :)

**Comment by mmoll** [comment]
also, needs a rebase.

File: plugins/ruby-foreman-tasks/debian/control (hunk 1 above)
Comment (ares) [reply plugins/ruby-foreman-tasks/debian/control]: yes :-) I'll fix that once 3.0.1 goes out, we need to fix a spec there to accept tasks 1.0.x

**Comment by ares** [comment]
rebased, fixed the wrong file and updated to 3.0.1

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
[test packaging]

**Comment by mmoll** [comment]
merged, díky @ares!

**Pull request merged**